extensions.json

```diff
@@ -1027,8 +1026,8 @@
     {
       "id": "streetsidesoftware.code-spell-checker",
       "repository": "https://github.com/streetsidesoftware/vscode-spell-checker",
-      "version": "1.9.2",
-      "checkout": "v1.9.2",
+      "version": "1.9.3-alpha.0",
+      "checkout": "master",
```

Comment:
The latest release tag still seems to be `v1.9.2`. Please undo this change.

Reply:
done!